clients/src/main/resources/common/message/DescribeQuorumResponse.json

```diff
@@ -17,7 +17,7 @@
   "apiKey": 55,
   "type": "response",
   "name": "DescribeQuorumResponse",
-  "validVersions": "0",
+  "validVersions": "0-1",
```

Comment:
Usually we add some comments to describe what has changed in each version. You can see the other request specs for examples.